p4proto/p4rt/p4_service.cc

```diff
@@ -389,6 +389,13 @@ ::grpc::Status P4Service::SetForwardingPipelineConfig(
                        node_id, "."));
     }
 
+    if (forwarding_pipeline_configs_ != nullptr &&
```

Comment:
Better to add comment indicating that it is safe check for now and later it will be taken care with proper fixes.
/*
  FIXME: xxx
 */

Reply:
Done, updated comments